server/middlewares/auth.js

```diff
@@ -50,4 +50,47 @@ const isAdmin = (req, res, next) => {
   });
 };
 
-module.exports = { isAuthorized, isAdmin };
+const userAuthorized = (req, res, next) => {
+  try {
+    verify(req.cookies.applicant, process.env.SECRET_KEY, (err, token) => {
+      if (err) {
+        res.status(401).send({
+          statusCode: 401,
+          auth: false,
+          error: 'you are Unauthorized',
+        });
+      } else {
+        res.status(200).json({
+          statusCode: 200,
+          auth: true,
+          error: 'you are authorized',
+        });
+      }
+    });
+  } catch (err) {
+    next(err);
+  }
+};
+
+const adminAuthorized = (req, res, next) => {
+  try {
+    verify(req.cookies.admin, process.env.SECRET_KEY, (err, token) => {
+      if (err) {
+        res.status(401).send({
+          statusCode: 401,
+          auth: false,
+          error: 'you are Unauthorized',
+        });
+      } else {
+        res.status(200).json({
+          statusCode: 200,
+          auth: true,
+          error: 'you are authorized',
```

Comment:
the same here

instead of send error to the user it's better to send a message
```suggestion
          message: 'you are authorized',
```